### PR TITLE
Improve CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -113,12 +126,13 @@ $short         = foo($bar);
 $long_variable = foo($baz);
 ```
```
Comment (mikaelrandy) [inline CONTRIBUTING.md]: I think it's a bad idea to write a code snippet to illustrate a bad exemple.
It's likely that only the code be read, and contributors could think they have to use this syntax

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -132,10 +146,10 @@ $foo = new $bar();
...
-# Arrays
+### Arrays
 
-Arrays should be formatted with a space separating each element (after the comma), and spaces around the `=>` key association operator, if applicable.
-If an array contains several values, put each of them on one line:
+Arrays **must** be formatted with a space separating each element (after the comma), and spaces around the `=>` key association operator, if applicable.
+If an array contains several values, they **must** be on their own lines:
 
 ```php
 $array = array(
```
Comment (mikaelrandy) [inline CONTRIBUTING.md]: Maybe explain, as atoum should support PHP 5.3, to avoid short syntax ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -145,11 +159,12 @@ $array = array(
...
 
-To get lenght of an array, use `sizeof` instead of `count` (no technical reason about that, it's just the current convention).
+To get the length of an array, `sizeof` **must** be used instead of `count` (no technical reason about that, it's just the current convention).
 
-# Quotes
+### Quotes
 
-You should always use single quote and always use a space between the dot and the concatenated parts to improve readability:
+* Single quotes **must** be used.
+* There *must** be a space between the dot and the concatenated parts to improve readability:
```
Comment (mikaelrandy) [inline CONTRIBUTING.md]: Little typo : "**must**"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,83 +1,93 @@
...
-All binary operators (operators that come between two values), such as `+`, `-`, `=`, `!=`, `==`, `>`, etc. should have a space before and after the operator, for readability. 
-For example, an assignment should be formatted as `$foo = $bar;` rather than `$foo=$bar;`. 
-Unary operators (operators that operate on only one value), such as `++`, should not have a space between the operator and the variable or number they are operating on.
+The key words "**must**", "**must not**", "**should**", "**should not**" and "**may**" in this document are to be interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt).
 
-# Casting
+### Indenting and Whitespace
 
-Put a space between the (type) and the `$variable` in a cast: `(int) $mynumber`.
+* Tabs **must** be used to indent lines.
```
Comment (Hywan) [inline CONTRIBUTING.md]:
```diff
- lines.
+ lines,
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,83 +1,93 @@
...
+* Lines **must not** have any trailing whitespace at the end.
```
Comment (Hywan) [inline CONTRIBUTING.md]:
```diff
- end.
+ end,
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,83 +1,93 @@
...
+* Files **must** be formatted with `\n` as the line ending (Unix line endings), not `\r\n` (Windows line endings).
```
Comment (Hywan) [inline CONTRIBUTING.md]:
```diff
- endings).
+ endings),
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,83 +1,93 @@
...
+* PHP files **must** be in UTF-8.
```
Comment (Hywan) [inline CONTRIBUTING.md]:
```diff
- UTF-8.
+ UTF-8,
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,83 +1,93 @@
...
+* PHP files **must** begin with `<?php ` and **should not** have an end tag (no `?>`).
```
Comment (Hywan) [inline CONTRIBUTING.md]: This one is correct!

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -183,26 +198,36 @@ This is a good example:
...
+Global variables **must not** be used.
 
-# Global Variables
+## Code of Conduct
 
-You should not use global variable.
+By contributing to atoum you agree to adopt the [Open CoC](http://todogroup.org/opencodeofconduct/#atoum/team@atoum.org):
 
-Thanks to the Drupal community for its work about its [coding convention](https://drupal.org/coding-standards).
+* Be friendly and patient
```
Comment (Hywan) [inline CONTRIBUTING.md]: Add commas at the end of all items except the latest which receive a dot.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jubianchi) [inline CONTRIBUTING.md]: I added a dedicated section: https://github.com/atoum/atoum/pull/543/files#diff-6a3371457528722a734f3c51d9238c13R83

**Comment by jubianchi** [comment]
@atoum/rms I'll let you review this PR and merge it if it's OK ;)

**Pull request merged**